src/client/pages/Register.tsx

```diff
@@ -1,108 +1,84 @@
+  public render() {
+    return (
+      <Scaffold view={View.LOGIN}>
+        <div className="page-title">
+          <Typography variant="h1" style={{ color: getColor(View.HOME) }}>
             {" "}
-            <br />
-            {"<< Go back to the Login page."}
-          </Link>
-        </Typography>
-      </div>
-      <div style={{ width: "80%", margin: "auto" }}>
-        <Grid container>
-          {/* <div className="pre-auth"> */}
-          {/* <form className="register-input-block" noValidate autoComplete="off"> */}
-          <Grid item xs={4}>
-            <div className="input-title">
-              {" "}
-              FRIST NAME <br />
-              <TextField
-                id="input-fName"
-                label=""
-                variant="outlined"
-                style={{ backgroundColor: "white" }}
-              />
-            </div>
-          </Grid>
-          <Grid item xs={4}>
-            <div className="input-title">
-              {" "}
-              LAST NAME <br />
-              <TextField
-                id="input-lName"
-                label=""
-                variant="outlined"
-                style={{ backgroundColor: "white" }}
-              />
-            </div>
-          </Grid>
-          <Grid item xs={4}>
-            <div className="input-title">
-              {" "}
-              BIRTHDAY <br />
-              <TextField
-                id="input-birthday"
-                label=""
-                variant="outlined"
-                style={{ backgroundColor: "white" }}
-              />
-            </div>
-          </Grid>
-          <Grid item xs={4}>
-            <div className="input-title">
-              {" "}
-              EMAIL ADDRESS <br />
-              <TextField
-                id="input-email"
-                label=""
-                variant="outlined"
-                style={{ backgroundColor: "white" }}
-              />
-            </div>
-          </Grid>
-          <Grid item xs={4}>
-            <div className="input-title">
-              {" "}
-              USERNAME <br />
-              <TextField
-                id="input-username"
-                label=""
-                variant="outlined"
-                style={{ backgroundColor: "white" }}
-              />
-            </div>
-          </Grid>
-          <Grid item xs={4}>
-            <div className="input-title">
-              {" "}
-              PASSWORD <br />
-              <TextField
-                id="input-password"
-                label=""
-                variant="outlined"
-                style={{ backgroundColor: "white" }}
-              />
-            </div>
-          </Grid>
-        </Grid>
-        {/* </form> */}
-        <Link className="login-button" href="/stories">
-          SIGN UP &gt;&gt;
-        </Link>
-      </div>
-      {/* Forgot password option needes */}
-    </Scaffold>
-  );
+            REGISTER{" "}
```

Comment:
Why are all of these spaces scattered around the code? Most (if not all) of them should be removed. The only time you need them is when you specifically want a space in the code. The reason for this is, spaces in HTML that are right next to a tag aren't considered a space. For example,
```
<div>
  Hello
</div>
```
doesn't insert a space between the `<div>` and the "Hello" even though there's a new line. If you do actually want a space between the `<div>` and the "Hello", you would need something like:
```
<div>
  {" "}Hello
</div>
```
Please update this on all other files too.

Reply:
The only thing is, those keep getting added in automatically by the code format tool, I've tried removing them but they keep coming back

Reply:
It might be a problem in the way I'm writing it, let me try